packages/logrotate/build

```diff
@@ -10,84 +10,20 @@ make
 make install
 
 
-config_file="$PKG_PATH/etc/logrotate_master.conf"
-mkdir -p $(dirname $config_file)
-cat <<'EOF' > "$config_file"
-compress
-delaycompress
-notifempty
-missingok
-nomail
+systemd_master="$PKG_PATH"/dcos.target.wants_master/dcos-logrotate-master.service
+mkdir -p "$(dirname "$systemd_master")"
+envsubst '$PKG_PATH' < /pkg/extra/dcos-logrotate-master.service > "$systemd_master"
 
-/var/lib/dcos/mesos/log/* {
-    olddir /var/lib/dcos/mesos/log/archive
-    maxsize 2000k
-    daily
-    rotate 7
-    copytruncate
-    postrotate
-        find  /var/lib/dcos/mesos/log/ /var/lib/dcos/mesos/log/archive -mtime +14 -delete
-    endscript
-}
-EOF
+systemd_slave="$PKG_PATH"/dcos.target.wants_slave/dcos-logrotate-agent.service
```

Comment:
Might as well name this variable `systemd_agent` since the service is called `dcos-logrotate-agent`, and it's only relevant in this one file.